packages/web/src/components/ui/Card/PackagesBySourceCard.tsx

```diff
@@ -17,7 +17,7 @@ const PackagesBySourceCard = ({
   packages,
   morePackagesCount,
 }: PackagesBySourceCardProps) => (
-  <Card title={sourceTitle} icon={sourceIcon}>
+  <Card title={sourceTitle} icon={sourceIcon} to={`/scan/${sourceTitle}`}>
```

Comment:
shouldn't it be `sourcePageUrl`?

Reply:
Not sure what you meant, but I'll probably split the title and the url into separate props. This component originally served two purposes: displaying packages by source scan and displaying packages by author.